nyc_taxi.py

```diff
@@ -0,0 +1,56 @@
+"""
+This program run the NYC_Taxi dataset through HTM.Core's anomaly detector and
+plots the results.
+"""
+from nab.corpus import DataFile
+from nab.detectors.htmcore.htmcore_detector import HtmcoreDetector
+import datetime
+import matplotlib.pyplot as plt
+import numpy as np
+
+def parse_timestamp(ts):
+    return datetime.datetime.strptime(ts, "%Y-%m-%d %H:%M:%S")
+
+srcPath = "data/realKnownCause/nyc_taxi.csv"
+dataSet = DataFile(srcPath)
+# dataSet.data = dataSet.data[0:1000]
+val     = dataSet.data["value"]
+ts      = dataSet.data["timestamp"]
+
+anomaly_windows = [
```

Comment:
what are these? manual ground-truth?

Reply:
Yes, I copied-pasted them...

Reply:
Im pretty sure I got these from the file:
labels/combined_windows.json

I could with a bit more effort have these labels be read in from that file, but since I also hardcoded which dataset it uses, I did not bother.

Reply:
I'd say it's ok to leave it working as-is now, just please add this comment that it's the GT and that it comes from file ...

Reply:
done, I added a comment.